lang/en-gb.xml

```diff
@@ -39,7 +39,7 @@
 		<application>Application</application>
 		<addcont>DLCs</addcont>
 		<license>License</license>
-		<save_data>Save Data</save_data>
+		<save_data>Saved Data</save_data>
```

Comment:
is save data 🤔

Reply:
for it i have no idea i need check more 💀 i have no set this tring, so i no know

Reply:
is not really writed in all lang, is only debug option, so english only
![image](https://user-images.githubusercontent.com/5261759/177781376-943bbc77-b978-4582-b5c2-a0db09a09164.png)